src/acl/Acl.cc

```diff
@@ -102,6 +102,18 @@ Acl::SetKey(SBuf &keyStorage, const char *keyParameterName, const char *newKey)
                         Here());
 }
 
+ScopedId
+Acl::ParsingContext::codeContextGist() const {
+    return ScopedId("ACL");
```

Comment:
Let's use an admin-friendlier spelling.

```suggestion
    return ScopedId("acl");
```